README.md

```diff
@@ -1,3 +1,54 @@
-# service-center [![Build Status](https://travis-ci.org/ServiceComb/service-center.svg?branch=master)](https://travis-ci.org/ServiceComb/service-center)
-A standalone service center to allow services to register their instance information and to discover providers of a given service
-EDIT1
+# Service Center [![Build Status](https://travis-ci.org/ServiceComb/service-center.svg?branch=master)](https://travis-ci.org/ServiceComb/service-center)
+
+A standalone Service Center to allow services to register their instance information and to discover providers of a given service
+
+- [http://www.servicecomb.io/service-center](http://www.servicecomb.io/service-center)
```

Comment:
this webpage is not ready. i suggest not putting the link here, or it may confuse readers.